src/UI/state/upload/uploadSlice.test.tsx

```diff
@@ -15,4 +19,65 @@ describe('uploadSlice', () => {
 
     expect(updatedState.modelFile).toEqual('file');
   });
+  it('updateValidationErrors', () => {
+    const error = [
+      {
+        key: 'Year',
+        row: 1,
+        errorType: 'Incorrect data type',
+        expectedType: 'number',
+        receivedType: 'string',
+      },
+      {
+        key: 'Latitude',
+        row: 1,
+        errorType: 'Required data',
+        expectedType: 'string',
+      },
+      {
+        key: 'Year',
+        row: 1,
+        errorType: 'Incorrect data type',
+        expectedType: 'number',
+        receivedType: 'string',
+      },
+      {
+        key: 'Latitude',
+        row: 1,
+        errorType: 'Required data',
+        expectedType: 'string',
+      },
```

Comment:
Worth sticking another row in here too, to test that they aren't all just stuck in the one row